src/modules/rover_ackermann/RoverAckermannGuidance/RoverAckermannGuidance.hpp

```diff
@@ -144,14 +141,14 @@ class RoverAckermannGuidance : public ModuleParams
 
 
 	MapProjection _global_ned_proj_ref{}; // Transform global to NED coordinates.
-	PurePursuit _pure_pursuit; // Pure pursuit library
+	PurePursuit _pure_pursuit{this}; // Pure pursuit library
```

Comment:
Just for curiosity: why did you have to construct it with `this` here?

Reply:
Maybe I'm wrong about this but the pure pursuit library now inherits from ModuleParams because of the parameters and for that I need this constructor:
`PurePursuit(ModuleParams *parent);`